src/core/agent/services/keriaNotificationService.ts

```diff
@@ -829,6 +827,8 @@ class KeriaNotificationService extends AgentService {
             oid: recordId,
           },
         });
+        const alias = new URL(operation.metadata?.oobi).searchParams.get("name") ?? randomSalt();
+        await this.props.signifyClient.contacts().update((operation.response as State).i, { alias });
```

Comment:
This shouldn't have moved. Please revert the last commit and read the comment. I'm saying you need to use the same `alias` from `connectionService.ts` and store it in the long running operation record.

Reply:
oh, you mean use alias in ConnectionRecord